wormhole-connect/src/utils/routes/cctpRelay.ts

```diff
@@ -156,26 +156,26 @@ export class CCTPRelayRoute extends CCTPManualRoute {
   ): Promise<boolean> {
     return false;
 
-    const sourceTokenConfig = TOKENS[sourceToken];
-    const destTokenConfig = TOKENS[destToken];
+    // const sourceTokenConfig = TOKENS[sourceToken];
+    // const destTokenConfig = TOKENS[destToken];
 
-    if (!sourceChain || !destChain || !sourceTokenConfig || !destTokenConfig)
-      return false;
+    // if (!sourceChain || !destChain || !sourceTokenConfig || !destTokenConfig)
+    //   return false;
 
-    const sourceChainName = wh.toChainName(sourceChain);
-    const destChainName = wh.toChainName(destChain);
+    // const sourceChainName = wh.toChainName(sourceChain);
+    // const destChainName = wh.toChainName(destChain);
 
-    if (sourceChainName === destChainName) return false;
+    // if (sourceChainName === destChainName) return false;
 
-    if (sourceTokenConfig.symbol !== CCTPTokenSymbol) return false;
-    if (destTokenConfig.symbol !== CCTPTokenSymbol) return false;
-    if (sourceTokenConfig.nativeNetwork !== sourceChainName) return false;
-    if (destTokenConfig.nativeNetwork !== destChainName) return false;
+    // if (sourceTokenConfig.symbol !== CCTPTokenSymbol) return false;
+    // if (destTokenConfig.symbol !== CCTPTokenSymbol) return false;
+    // if (sourceTokenConfig.nativeNetwork !== sourceChainName) return false;
+    // if (destTokenConfig.nativeNetwork !== destChainName) return false;
 
-    return (
-      CCTPRelay_CHAINS.includes(sourceChainName) &&
-      CCTPRelay_CHAINS.includes(destChainName)
-    );
+    // return (
+    //   CCTPRelay_CHAINS.includes(sourceChainName) &&
+    //   CCTPRelay_CHAINS.includes(destChainName)
+    // );
```

Comment:
Why are these lines commented out?

Reply:
Unreachable code, commented them to fix the lint, but now this needs to be rebased 🙂